README.md

```diff
@@ -112,22 +112,27 @@ Consider trying out and providing feedback on the following:
 - [GitOps as an add-on](https://github.com/Azure/AKS/issues/1967)
 - [Azure AD Pod Identity as an add-on](https://docs.microsoft.com/azure/aks/use-azure-ad-pod-identity)
 
+## Related Reference Implementations
+
+The AKS Baseline was used as the foundation for the following additional reference implementations. These build on the learnins of the AKS Baseline and applies a specific lens to the cluster to align a specific topology, requirement, and/or workload type.
+
+- [AKS Baseline for Multi-Region Clusters](https://github.com/mspnp/aks-baseline-multi-region)
+- [AKS Baseline for Regulated Workloads](https://github.com/mspnp/aks-baseline-regulated)
+- [AKS Baseline for Microservices](https://github.com/mspnp/aks-fabrikam-dronedelivery)
```

Comment:
/sight is aks-fabriakm-dronedelivery the repo name for this now we got authoritative RA on top of this and the workload got surgically removed 😸 ?

anyhow, nothing to be done as a part of this PR - resolving it now.